Component/MainPage.js

```diff
@@ -0,0 +1,106 @@
+// learn how to display your input as a list - the money that people spend  
+// get the input of daily expenses and list them below
+// do calculation with those expenses and categorize them 
+
+
+import React, { Component } from 'react';
+import {useState} from 'react';
+import { View,StyleSheet,Text,Image,TextInput,FlatList} from 'react-native';
+import style from 'react-native-datepicker/style';
+
+
+const Input = (props,{navigate}) => {
+  const[money,Setmoney]= useState('');
+
+    return (
+      // Try setting `flexDirection` to `column`.
+      <View style={styles.main}>
+        <Text style = {styles.Text}> Enter the expenses:</Text>
+         <TextInput style = {styles.money}       
+            placeholder= 'Enter your expenses'
+            autoCapitalize='none'
+            keyboardType='numeric'
+            placeholderTextColor ='gray'
+            onChangeText={(val) => Setmoney(val) }/> 
+            <Text style={styles.Text}>
+               expenses : {money} 
+               </Text>
+               <FlatList
+        data={[
+          {key: 'Devin'},
+          {key: 'Dan'},
+          {key: 'Dominic'},
+          {key: 'Jackson'},
+          {key: 'James'},
+          {key: 'Joel'},
+          {key: 'John'},
+          {key: 'Jillian'},
+          {key: 'Jimmy'},
+          {key: 'Julie'},
```

Comment:
For hard coded data like this, a good practice is to import this from another file in your project like 'data.js' or something

Reply:
yes you right i was just trying to see if it worked first cuz i still didnt managed to but totally agree